tests/behavioural/features/steps/secure_messaging_context_helper.py

```diff
@@ -63,7 +62,7 @@ class SecureMessagingContextHelper:
     def __init__(self):
         self._token_data = {}
         self._headers = {'Content-Type': 'application/json', 'Authorization': ''}
-        self.token_data = SecureMessagingContextHelper.__INTERNAL_BRES_USER_TOKEN  # use attribute to set headers
+        self.token_data = SecureMessagingContextHelper.__INTERNAL_SPECIFIC_USER_TOKEN  # use attribute to set headers
```

Comment:
There is still some references to `__INTERNAL_BRES_USER_TOKEN` fragmented around in some places here